scripts/deployUnmockedColonyVersions.js

```diff
@@ -0,0 +1,72 @@
+// Input:
+/* globals artifacts */
+
+// Check out v3
+// Deploy v3 resolver
+// Point v3 on network to that resolver
+
+// Checkout out v4
+// Deploy v4 resolver
+// Point v4 on network to that resolver
+const util = require("util");
+const exec = util.promisify(require("child_process").exec);
+const cnAddress = require("../etherrouter-address.json").etherRouterAddress; // eslint-disable-line import/no-unresolved
+
+const IColonyNetwork = artifacts.require("./IColonyNetwork");
+const IMetaColony = artifacts.require("./IMetaColony");
+
+module.exports = async () => {
+  // While debugging, always checkout currenthash
+  // await exec("git checkout 2690fba3d3002fd72912cd74f1fbf4932c734e94 && git submodule update");
+  const currentHash = await exec("git log -1 --format='%H'");
+  // const currentHash = res.stdout.trim();
+  let v3ResolverAddress;
+  let v4ResolverAddress;
+  const colonyNetwork = await IColonyNetwork.at(cnAddress);
+  const metaColonyAddress = await colonyNetwork.getMetaColony();
+  const metaColony = await IMetaColony.at(metaColonyAddress);
+  try {
+    await exec("mv ./build ./buildBackup");
+    await exec("git checkout ad5569de24567517aa12624e29600c9136fb594d && git submodule update");
```

Comment:
Does this version not have a release tag? That's too bad.

Reply:
It does, I am just bad :smile: